rnd/autogpt_builder/src/components/edit/control/BlocksControl.tsx

```diff
@@ -36,8 +36,12 @@ export const BlocksControl: React.FC<BlocksControlProps> = ({
 }) => {
   const [searchQuery, setSearchQuery] = useState("");
 
-  const filteredBlocks = blocks.filter((block: Block) =>
-    block.name.toLowerCase().includes(searchQuery.toLowerCase()),
+  const filteredBlocks = blocks.filter(
+    (block: Block) =>
+      block.name.toLowerCase().includes(searchQuery.toLowerCase()) ||
+      beautifyString(block.name)
+        .toLowerCase()
+        .includes(searchQuery.toLowerCase()),
```

Comment:
Indirectly related: block is not searchable using its beatified name. e.g: I can't find the GetOpenWeatherMapBlock using `weather map` query.